src/views/image/TemplateZones.vue

```diff
@@ -110,6 +101,24 @@
         </a-form>
       </a-spin>
     </a-modal>
+
+    <a-modal
+      v-if="'deleteTemplate' in $store.getters.apis"
```

Comment:
Adding `v-if` in modal itself will make delete button visible irrespective user access.
It is showing for read-only admin user
![Screenshot from 2020-06-12 16-13-44](https://user-images.githubusercontent.com/153340/84494891-e180b780-acc7-11ea-84ed-c25ee3c7f8a0.png)